cmd/namespace_test.go

```diff
@@ -0,0 +1,129 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+
+	"github.com/appuio/seiso/cfg"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_validateNamespaceCommandInput(t *testing.T) {
+	type args struct {
+		args   []string
+		config cfg.Configuration
+	}
+	tests := []struct {
+		name    string
+		input   args
+		wantErr bool
+	}{
+		{
+			name: "ShouldThrowError_IfNoLabelSelector",
```

Comment:
Could also follow the map[string]struct {} syntax.
```suggestion
	tests := map[string]struct {
		args []string
		config cfg.Configuration
		wantErr bool
	}{
		"ShouldThrowError_IfNoLabelSelector": {
```